__tests__/endpoints.test.js

```diff
@@ -35,3 +36,25 @@ describe("/api/categories", () => {
     return request(app).get("/api/categor").expect(404);
   });
 });
+describe("/api", () => {
+  test("GET - status 200, responds with right JSON object", () => {
+    const checkData = fsSync.readFileSync("./endpoints.json", "utf-8");
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const responseData = response.body.api;
+        expect(responseData).toEqual(checkData);
```

Comment:
Points for improvement: 
You could iterate through the returned array and be more specific about what form the JSON object should take.

Maybe have a look at [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject)